alacritty/src/event.rs

```diff
@@ -1248,7 +1248,11 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                             let preedit = if text.is_empty() {
                                 None
                             } else {
-                                Some(Preedit::new(text, cursor_offset.map(|offset| offset.0)))
+                                // TODO remove with winit update.
```

Comment:
I thought you weren't sure if winit will get updated at all, since this is already fixed?

This sounds like it will be?

Reply:
I mean there will be an `0.28` update, just not soon.

Reply:
What speaks against a patch release? Considering this affects **every** winit user.

Reply:
It's a very niche case only on Wayland with old bugged software. I just don't have enough patches to make a release. It'll like this oneliner one and that's about it...

Reply:
I mean if it's reason enough to patch Alacritty, isn't it reason enough to release a winit patch?

Reply:
PepeSad.